source/misc/fish_startup.in

```diff
@@ -1,71 +1,70 @@
-if [ x"$TERM" != "xscreen" ] 
-  function iterm2_status
-    printf "\033]133;D;%s\007" $argv
-  end
+set -l fishversion (echo $FISH_VERSION | sed 's|\(.\)\.\(.\).*|\1\2|')
+if [ $fishversion -ge 22 ]
```

Comment:
Please put the old code in an else clause. I'll take it out after a while, but we can't count on everyone having the newest version for a while.